### PR TITLE
Replace debounce with retries

### PR DESCRIPTION
**Pull request by elainearbaugh**
still need to test on local environment + stage

### CODE REVIEW

File: cabot/cabotapp/migrations/0025_auto__del_field_statuscheck_debounce__add_field_statuscheck_retries.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Renaming field 'StatusCheck.debounce'
+        db.rename_column(u'cabotapp_statuscheck', 'debounce', 'retries')
```
Comment (mahendra) [inline cabot/cabotapp/migrations/0025_auto__del_field_statuscheck_debounce__add_field_statuscheck_retries.py]: 👍

**Comment by elainearbaugh** [comment]
tested on stage:

- Create new check with retries
- Edit a check and change retries
- Change retries and make sure the service fails after that number of failures
- Make sure retries for each check is the same as the old debounce

**Comment by mahendra** [comment]
looks good to go to prod. today?

**Comment by elainearbaugh** [comment]
@mahendra sure!!

**Pull request merged**